thanos/charts/templates/tests/test-thanos-config.yaml

```diff
@@ -62,4 +62,11 @@ data:
       verify "there is 1 servicemonitor named '{{ .Release.Name }}'"
     }
     {{ end }}
+
+    {{ if and (not .Values.thanos.query.standalone) (not .Values.ci.enabled) }}
```

Comment:
You will be able to test this use case once this issue is done https://github.com/cloudoperators/greenhouse-extensions/issues/615 :)